doc/rados/operations/placement-groups.rst

```diff
@@ -676,4 +676,4 @@ entirely. To mark the "unfound" objects as "lost", execute the following::
 
 .. _Create a Pool: ../pools#createpool
 .. _Mapping PGs to OSDs: ../../../architecture#mapping-pgs-to-osds
-.. _pgcalc: http://ceph.com/pgcalc/
+.. _pgcalc: https://old.ceph.com/pgcalc/
```

Comment:
Cool, I didn't even know this was a thing!

Reply:
Yeah. Over the next year, I've got to improve the exposure of stuff like this.